react-forms/my-app/src/RegistrationFormControlled.tsx

```diff
@@ -0,0 +1,38 @@
+import { FormEvent, useState } from 'react';
+export function RegistrationFormControlled() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  function handleSubmit(event: FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    console.log({ username, password });
```

Comment:
If you want to clear your form after submitting like you did for your uncontrolled form, you would call both your setter functions with an empty string. Good job! that's a good habit.
 eg. setUsername("")